language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SnippetGenerator.java

```diff
@@ -459,6 +459,17 @@ public static SnippetBlock getFunctionDefSnippet() {
                 ItemResolverConstants.SNIPPET_TYPE, Kind.SNIPPET);
     }
 
+    /**
+     * Get expression bodied function snippet block.
+     *
+     * @return {@link SnippetBlock} Generated snippet
+     */
+    public static SnippetBlock getExpressionBodiedFunctionDefSnippet() {
+        String snippet = "function ${1:name}(${2})${3} => ${4}";
```

Comment:
Is there any reason why we're not including `();` in the generated snippet? 
What if we suggest `function name() => ();` in the snippet?

Reply:
As per the spec, there can be any expression after the `=>`. Having `()` is not a must.
```
function-defn := 
   metadata
   [public] function-quals
   function identifier function-signature function-defn-body

function-defn-body :=
   block-function-body
   | expr-function-body ;
   | external-function-body ;

expr-function-body := => expression
```

Reply:
Do we have some other incomplete snippet suggestion too? Can't we think `()` as the default expression since the return type of this function is also `()`. In snippet suggestions like `if true {}` also uses default values right?

Reply:
Also given that the 'design' code lens is provided for these expression bodied functions, we are expecting some novice Ballerina users to use this snippet completion to get started with the visual data mapper.
In that case we thought to have the `();` part since that shows the entire base structure.
@hevayo @kaviththiranga WDYT?

Reply:
@madushajg Had a discussion with @TharushiJay as well. Agreed with both of you. I will update the snippet to include the `();` part.